### PR TITLE
Group page proxy fields

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/contacts/views.py [hunk 1]
```diff
@@ -632,7 +632,11 @@ def get_context_data(self, *args, **kwargs):
             org = self.request.org
 
             context["current_group"] = self.group
-            context["contact_fields"] = ContactField.get_fields(org).order_by("-priority", "id")
+
+            fields = ContactField.get_fields(org).order_by("-priority", "id")
+            proxy_fields = org.fields.filter(key__in=("last_seen_on", "created_on"), is_proxy=True).order_by("-key")
+            context["contact_fields"] = list(fields) + list(proxy_fields)
+
```
Comment (norkans7) [inline temba/contacts/views.py:639]: I think we can include the last seen on and created_on fields as well on the group pages

**Comment by rowanseymour** [comment]
@norkans7 can we get a screenshot perhaps to understand what is being fixed here?

File: temba/contacts/views.py (hunk 1 above)
Comment (rowanseymour) [reply temba/contacts/views.py:639]: Weird.. has it always been the case that we show all fields on group pages or did I break that??

Comment (norkans7) [reply temba/contacts/views.py:639]: No we just have never shown those fields

**Pull request merged**